packages/snaps-execution-environments/webpack.config.js

```diff
@@ -38,45 +66,14 @@ module.exports = (_, argv) => {
               'dist',
               'lockdown.umd.min.js',
             ),
-            to: path.resolve(ENVIRONMENTS, 'node-process/lockdown.umd.min.js'),
-            toType: 'file',
-          },
-          {
-            from: path.resolve(
-              `${path.dirname(require.resolve('ses/package.json'))}`,
-              'dist',
-              'lockdown.umd.min.js',
-            ),
-            to: path.resolve(ENVIRONMENTS, 'node-thread/lockdown.umd.min.js'),
+            to: path.resolve(ENVIRONMENTS, 'lockdown.umd.min.js'),
```

Comment:
I've reverted this, since I realised that this adds complexity (especially when moving the lockdown files with the iframe). We can reconsider this in the future.